### DISCUSSION TITLE
Deep Link to one of verticals on news archive

### DISCUSSION THREAD

**Discussion by damianino1** [microsoft-search/pnp-modern-search#1972] [Q&A]
Hello,

would it be possible to access directly to one of verticals on news archive? If yes how to do it? How to define the deeplink?

![image](https://user-images.githubusercontent.com/102507233/160341177-1281382e-9790-4d58-8ceb-4575f5770dd1.png)

**Comment by FranckyC** [comment]
You can use the default `v` query string parameter for that:

![image](https://user-images.githubusercontent.com/5659122/160649103-fccc32c0-01f8-4a7e-a3de-605090cc370e.png)


Usage: `?v=HR News`
